swap/swap.go

```diff
@@ -344,6 +352,36 @@ func (s *Swap) handleEmitChequeMsg(ctx context.Context, p *Peer, msg *EmitCheque
 	return err
 }
 
+func (s *Swap) computeAvailableBalance() (uint64, error) {
+	// get the total amount (Wei) ever deposited
+	deposit, err := s.contract.TotalDeposit()
+	if err != nil {
+		return 0, err
+	}
+	// get the total amount (Wei) ever withdrawn
+	withdrawn, err := s.contract.TotalWithdrawn()
+	if err != nil {
+		return 0, err
+	}
+	// read the total value (Wei) of all cheques cheques sent
+	var totalPaidOut uint64
+	sentCheques, err := s.SentCheques()
+	if err != nil {
+		return 0, err
+	}
+	for _, v := range sentCheques {
```

Comment:
we can maybe call this something other than `v` for greater clarity (e.g. `sentCheque`)